concourse/vars-files/gpg-keys.yml

```diff
@@ -532,4 +546,19 @@ gpg_public_keys:
   kOSyK5KJF+3+fBXkx7dkNVhra07mES3NTMMK4LB7EbNSAxKM+DPLVHhbIT/qcOp9
   MxWm/YwTPjuPPnb7YSleT6T6tkfjE4FuFN+rwtfdq/QzfeUq8g68J906
   =OOfo
-  -----END PGP PUBLIC KEY BLOCK-----
\ No newline at end of file
+  -----END PGP PUBLIC KEY BLOCK-----
+- |
+  -----BEGIN PGP PUBLIC KEY BLOCK-----
+
+  mDMEZeHVchYJKwYBBAHaRw8BAQdA+ib5l0tK5Nabur1hrRsV1ChL0o3MB6J97qgJ
```

Comment:
This seems shorter than the others, should it be longer?

Reply:
true, it seemed short to me as well but seems to work nevertheless.

Reply:
I double checked by exporting my public key again and it matches what's here, I believe it's shorter as I'm using ed25519 rather than rsa